2025-03-30 21:29:17,603 - INFO - Succesfully read the YAML file
2025-03-30 21:29:17,603 - INFO - Data Ingestion started ....
2025-03-30 21:30:12,641 - INFO - Succesfully read the YAML file
2025-03-30 21:30:12,641 - INFO - Data Ingestion started ....
2025-03-30 21:30:12,641 - INFO - Starting Data ingestion process from GCP
2025-03-30 21:30:17,766 - WARNING - No project ID could be determined. Consider running `gcloud config set project` or setting the GOOGLE_CLOUD_PROJECT environment variable
2025-03-30 21:30:17,766 - ERROR - Error while downloading data from GCP
2025-03-30 21:30:17,766 - ERROR - Error in D:\ML_Ops\ML_ops_project2\src\data_ingestion.py , line 24: Failed to download data 
2025-03-30 21:30:17,766 - INFO - Data Ingestion Done .....
2025-03-30 21:47:54,806 - INFO - Succesfully read the YAML file
2025-03-30 21:47:54,807 - INFO - Data Ingestion started ....
2025-03-30 21:47:54,807 - INFO - Starting Data ingestion process from GCP
2025-03-30 21:47:58,841 - ERROR - Error while downloading data from GCP
2025-03-30 21:47:58,841 - ERROR - Error in D:\ML_Ops\ML_ops_project2\src\data_ingestion.py , line 41: Failed to download data 
2025-03-30 21:47:58,841 - INFO - Data Ingestion Done .....
2025-03-30 21:53:28,974 - INFO - Succesfully read the YAML file
2025-03-30 21:53:28,975 - INFO - Data Ingestion started ....
2025-03-30 21:53:28,975 - INFO - Starting Data ingestion process from GCP
2025-03-30 21:53:31,355 - ERROR - Error while downloading data from GCP
2025-03-30 21:53:31,357 - ERROR - Error in D:\ML_Ops\ML_ops_project2\src\data_ingestion.py , line 41: Failed to download data 
2025-03-30 21:53:31,357 - INFO - Data Ingestion Done .....
2025-03-30 21:56:11,017 - INFO - Succesfully read the YAML file
2025-03-30 21:56:11,018 - INFO - Data Ingestion started ....
2025-03-30 21:56:11,018 - INFO - Starting Data ingestion process from GCP
2025-03-30 21:56:13,656 - ERROR - Error while downloading data from GCP
2025-03-30 21:56:13,657 - ERROR - Error in D:\ML_Ops\ML_ops_project2\src\data_ingestion.py , line 40: Failed to download data
2025-03-30 21:56:13,657 - INFO - Data Ingestion Done .....
2025-03-30 22:00:20,192 - INFO - Succesfully read the YAML file
2025-03-30 22:00:20,193 - INFO - Data Ingestion started ....
2025-03-30 22:00:20,193 - INFO - Starting Data ingestion process from GCP
2025-03-30 22:00:21,370 - WARNING - No project ID could be determined. Consider running `gcloud config set project` or setting the GOOGLE_CLOUD_PROJECT environment variable
2025-03-30 22:00:21,370 - ERROR - Error while downloading data from GCP
2025-03-30 22:00:21,370 - ERROR - Error in D:\ML_Ops\ML_ops_project2\src\data_ingestion.py , line 25: Failed to download data
2025-03-30 22:00:21,370 - INFO - Data Ingestion Done .....
2025-03-30 22:00:31,825 - INFO - Succesfully read the YAML file
2025-03-30 22:00:31,826 - INFO - Data Ingestion started ....
2025-03-30 22:00:31,826 - INFO - Starting Data ingestion process from GCP
2025-03-30 22:00:34,602 - ERROR - Error while downloading data from GCP
2025-03-30 22:00:34,602 - ERROR - Error in D:\ML_Ops\ML_ops_project2\src\data_ingestion.py , line 40: Failed to download data
2025-03-30 22:00:34,602 - INFO - Data Ingestion Done .....
2025-03-30 22:00:48,425 - INFO - Succesfully read the YAML file
2025-03-30 22:00:48,425 - INFO - Data Ingestion started ....
2025-03-30 22:00:48,425 - INFO - Starting Data ingestion process from GCP
2025-03-30 22:00:50,589 - ERROR - Error while downloading data from GCP
2025-03-30 22:00:50,589 - ERROR - Error in D:\ML_Ops\ML_ops_project2\src\data_ingestion.py , line 40: Failed to download data
2025-03-30 22:00:50,590 - INFO - Data Ingestion Done .....
2025-03-30 22:01:14,395 - INFO - Succesfully read the YAML file
2025-03-30 22:01:14,395 - INFO - Data Ingestion started ....
2025-03-30 22:01:14,395 - INFO - Starting Data ingestion process from GCP
2025-03-30 22:01:16,452 - ERROR - Error while downloading data from GCP
2025-03-30 22:01:16,452 - ERROR - Error in D:\ML_Ops\ML_ops_project2\src\data_ingestion.py , line 40: Failed to download data
2025-03-30 22:01:16,454 - INFO - Data Ingestion Done .....
2025-03-30 22:07:07,165 - INFO - Succesfully read the YAML file
2025-03-30 22:07:07,166 - INFO - Data Ingestion started ....
2025-03-30 22:07:07,166 - INFO - Starting Data ingestion process from GCP
2025-03-30 22:07:08,290 - ERROR - Error while downloading data from GCP
2025-03-30 22:07:08,290 - ERROR - Error in D:\ML_Ops\ML_ops_project2\src\data_ingestion.py , line 40: Failed to download data
2025-03-30 22:07:08,290 - INFO - Data Ingestion Done .....
2025-03-30 22:08:20,966 - INFO - Succesfully read the YAML file
2025-03-30 22:08:20,966 - INFO - Data Ingestion started ....
2025-03-30 22:08:20,966 - INFO - Starting Data ingestion process from GCP
2025-03-30 22:08:21,977 - ERROR - Error while downloading data from GCP: 404 GET https://storage.googleapis.com/download/storage/v1/b/mlops_project_2_anime_recommendation/o/animme.csv?alt=media: No such object: mlops_project_2_anime_recommendation/animme.csv: ('Request failed with status code', 404, 'Expected one of', <HTTPStatus.OK: 200>, <HTTPStatus.PARTIAL_CONTENT: 206>)
2025-03-30 22:08:21,977 - ERROR - Error in D:\ML_Ops\ML_ops_project2\src\data_ingestion.py , line 40: Failed to download data
2025-03-30 22:08:21,977 - INFO - Data Ingestion Done .....
2025-03-30 22:11:58,598 - INFO - Succesfully read the YAML file
2025-03-30 22:11:58,599 - INFO - Data Ingestion started ....
2025-03-30 22:11:58,599 - INFO - Starting Data ingestion process from GCP
2025-03-30 22:12:01,908 - INFO - Downloading Smaller Files ie anime and anime_with synopsis
2025-03-30 22:12:04,634 - INFO - Downloading Smaller Files ie anime and anime_with synopsis
2025-03-30 22:16:03,080 - INFO - Large file detected Only downloading 5M rows
2025-03-30 22:16:03,105 - INFO - Data Ingestion Completed ....
2025-03-30 22:16:03,105 - INFO - Data Ingestion Done .....
